definitions/features/hammer.rb

```diff
@@ -7,7 +7,7 @@ class Features::Hammer < ForemanMaintain::Feature
     label :hammer
     confine do
       # FIXME: How does this run on proxy?
```

Comment:
We can answer this: it can run on a proxy, but doesn't by default. Probably safe to remove.